### DISCUSSION TITLE
Retrieving IPFS doc data using Python

### DISCUSSION THREAD

**Discussion by AryanSai** [smartcontractkit/full-blockchain-solidity-course-py#1707] [Q&A] [answered]
Can someone help me in reading an ipfs doc using some python api... Thanks in advance

**Comment by cromewar** [comment, marked as answer]
Hello @AryanSai 
Does [this](https://towardsdatascience.com/store-unlimited-data-for-free-on-ipfs-and-retrieve-it-with-python-8db7297b493d) help? I a comprehensive article about it.
Cheers.

**Reply by AryanSai** [reply]
The webpage is not available.

**Reply by cromewar** [reply]
sorry corrected.

**Reply by AryanSai** [reply]
Thanks!!
